doc/cla/individual/OsirisRoman.md

```diff
@@ -0,0 +1,11 @@
+Ecuador, 2020-30-04
```

Comment:
Thanks for the signature ! Could you keep the name of the file lowercase?

Reply:
ok. I'll do it.